src/hostname/hostnamed.c

```diff
@@ -1560,7 +1583,8 @@ static int build_describe_response(Context *c, bool privileged, sd_json_variant
                         SD_JSON_BUILD_PAIR_CONDITION(!sd_id128_is_null(product_uuid), "ProductUUID", SD_JSON_BUILD_ID128(product_uuid)),
                         SD_JSON_BUILD_PAIR_CONDITION(sd_id128_is_null(product_uuid), "ProductUUID", SD_JSON_BUILD_NULL),
                         SD_JSON_BUILD_PAIR_CONDITION(local_cid != VMADDR_CID_ANY, "VSockCID", SD_JSON_BUILD_UNSIGNED(local_cid)),
-                        SD_JSON_BUILD_PAIR_CONDITION(local_cid == VMADDR_CID_ANY, "VSockCID", SD_JSON_BUILD_NULL));
+                        SD_JSON_BUILD_PAIR_CONDITION(local_cid == VMADDR_CID_ANY, "VSockCID", SD_JSON_BUILD_NULL),
+                        SD_JSON_BUILD_PAIR("ChassisAssetTag", SD_JSON_BUILD_STRING(chassis_asset_tag)));
```

Comment:
_BUILD_PAIR_STRING

Maybe place this closer to Hardware*?

Reply:
Want me to also clean up the rest in a separate commit?

Reply:
That would be lovely :)